package.json

```diff
@@ -53,13 +53,13 @@
   ],
   "version": "2.0.0-dev",
   "dependencies": {
-    "tar": "^7.2.0"
+    "tar": "^7.4.0"
```

Comment:
Does this need to be changed? All other changes in the PR are either lock file updates or dev dependency updates. Both of which would constitute adding the `changelog:skip` label as they have no impact on an end user. This changes the version requirement of a dependency which does impact the end user (although only for those installing from npm) which should be reflected in the title so it appears in release notes.

Reply:
ok, I'll remove this from the PR.  We can do a separate one for the user impacting dependency updates.